src/webapp/src/client/app/kudos/kudosify-modal/kudosify-modal.html

```diff
@@ -29,6 +29,19 @@ <h3 class="modal-title" translate="kudos.submitKudosForAchievement"></h3>
             <label for="choose-type" translate="kudos.chooseType"></label>
             <div class="kudos-modal-buttons-group kudos-points-type">
                 <button class="btn btn-sm btn-info kudos-type-button kudos-points-type kudosify-modal-buttons-inactive"
+                        ng-if="(!kudosType.isActive && vm.isAdmin)"
```

Comment:
I don't see this check anywhere on the backend side. Is that true or might it be I missed something?
Currently it seems with minor change in UI/request you can still use inactive Kudos type..

Reply:
Correct, I should add this verification in API as well